news.Rmd

```diff
@@ -2,12 +2,37 @@
 
 Each user-facing change to a package should be accompanied by a bullet in `NEWS.md`. Minor changes to documentation don't need to be documented, but it's worthwhile to draw attention to sweeping changes and to new vignettes.
 
-## Bullets
+## In-development
 
 The goal of the bullet is to briefly describe the change so users of the packages can understand what's changed. This can be similar to the commit message, but written with a user (not developer) in mind.
```

Comment:
```suggestion
The goal of the bullet is to briefly describe the change so users of the packages can understand what's changed. This can be similar to the commit message, but written with a user (not developer) in mind. It's worth emphasizing this point -- the reader of your NEWS entries is likely unfamiliar with the day-to-day development work or internals of your package. Think carefully about how to summarize what's changed and why it matters for them.
```

Reply:
I like it. I added another sentence to reinforce that if there isn't anything user facing, you don't need a bullet.